investigate_water_ops.R

```diff
@@ -0,0 +1,57 @@
+# data from f_clean_inundation
+
```

Comment:
Need a read.csv or load data of some sort...can't duplicate this code without the dataset?